site/public/topvis.js

```diff
@@ -0,0 +1,3 @@
+/*! For license information please see topvis.js.LICENSE.txt */
```

Comment:
Do we really want this file in the repo? It is generated automatically on build and should be copied to the static site dir in the corresponding build step. Otherwise we may have version conflicts and build failures